web/client/components/widgets/builder/wizard/map/enhancers/__tests__/handleMapSelect-test.jsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const React = require('react');
+const ReactDOM = require('react-dom');
+const {createSink} = require('recompose');
+const expect = require('expect');
+import handleMapSelect from "../handleMapSelect";
```

Comment:
```suggestion
import React from 'react';
import ReactDOM from 'react-dom';
import {createSink} from 'recompose';
import expect from 'expect';
import handleMapSelect from "../handleMapSelect";
```
Mixed require and import are not allowed. When you find some require, please replace them with import (and module.exports with export or export default).
(when you rewrite, take into account all the modules that use require on that file, because export default needs `require(...).default`. 

We are going to remove all requires in the next release (as we start with 2019.03.xx). 

**note about testing**:
You can run a single test by editing tests.webpack.js
```
var context = require.context('../web/client/components/widgets/builder/wizard/map/', true, /(-test\.jsx?)|(-test-chrome\.jsx?)$/);
context.keys().forEach(context);
module.exports = context;
```
This way you can edit only files in the directory. To make this enhancer testable we need an import more (see my suggestion in handleMapSelect.js :

import '../../../../../../libs/bindings/rxjsRecompose';

This import set up rxjs + Recompose, if missing. It's ok to run this code once. If you're running a single import, it have to be included (we will refactor a little bit to avoid this requirement in the future)